### PR TITLE
PLAT-1814: Add shadow variant to card

### PR DESCRIPTION
**Pull request by mihaelpalfi**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1814)

#### Description
As per JIRA ticket

image

#### ChangeLOG

[Changelog](https://zen-ui.zengrc.com/?path=/docs/changelog--page)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: src/components/zen-card/zen-card.stories.mdx [hunk 1]
```diff
@@ -10,9 +10,36 @@ const argTypes = getArgTypes(compData);
 
 <Meta title="Containers/Card" component="zen-card" argTypes={argTypes} />
 
+export const Story1 = () => {
+  return html/*html*/ `
+    <sb-zen-space vertical spacing="lg" horizontal-align="center">
+      <sb-zen-text bold>Default variant</sb-zen-text>
+      <sb-zen-card>
+        <sb-zen-text>
+          Elementum rutrum sapien faucibus netus quisque eu ornare pulvinar dui, ipsum scelerisque volutpat ullamcorper
```
Comment (chavlji) [inline src/components/zen-card/zen-card.stories.mdx:19]: NIT: Would be nice to put this long repeating text into some var.

**Review by chavlji — APPROVED** [review]
Nice!

File: src/components/zen-card/zen-card.stories.mdx (hunk 1 above)
Comment (mihaelpalfi) [reply src/components/zen-card/zen-card.stories.mdx]: Jup. Done.

**Pull request merged**